lib/core/src/Cardano/Pool/Ranking.hs

```diff
@@ -66,14 +66,14 @@ module Cardano.Pool.Ranking
     (
       -- * Formulas
       desirability
+    , saturation
 
     , saturatedPoolRewards
     , saturatedPoolSize
 
       -- * Types
     , EpochConstants (..)
     , Pool (..)
-    , Lovelace (..)
```

Comment:
My thought was to keep this module completely independent on any other parts in `cardano-wallet`. Hence the `Lovelace`-type, instead of `Data.Quantity`.

Reply:
`Data.Quantity` has nothing to do with the wallet, really. So this still holds. We probably need to move these helpers like `Ratio` and `Positive` in some dedicated places at some point to, like `Data.Quantity`.